langgraph_bot/nodes/tnode/generate_image_node.py

```
@@ -0,0 +1,39 @@
+    try:
+        secret_key = os.getenv("POLLINATIONS_SECRET_KEY")
+    except Exception as e:
+        print(f"Error fetching Pollinations Secret Key: {e}")
+        return {}
+    summary = state.get("summary") or state.get("data")
+    
+    prompt_image = IMAGE_GENERATION_PROMPT.format(summary=summary)
+
+    #API call to Pollinations for image generation
+    try:
+        response = requests.get(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)